helm/auto-deployment/bold-bi-auto-deployment-using-helm.md

```diff
@@ -0,0 +1,439 @@
+# Bold BI Auto Deployment using Helm
+This section helps you to deploy [Bold BI](https://www.boldbi.com/) using helm chart without manually activating license and configuring the startup from browser. You can pass the license, startup and branding related customization as environment variable in values.yaml file.
+
+## Deployment prerequisites
+
+* [Install Helm](https://helm.sh/docs/intro/install/) to deploy Bold BI using Helm.
+* [File Storage](../docs/pre-requisites.md#file-storage)
+* [Create and connect a cluster](../docs/pre-requisites.md#create-a-cluster)
+* [Load Balancing](../docs/pre-requisites.md#load-balancing)
+
+**Note:** Note the [Ingress IP address](docs/pre-requisites.md#get-ingress-ip) and map it with a DNS to crafting values.yaml when installing Bold BI with helm chart.
+
+## Get Repo Info
+
+1. Add the Bold BI helm repository
+
+```console
+helm repo add boldbi https://boldbi.github.io/boldbi-kubernetes
+helm repo update
+```
+
+2. View charts in repo
+
+```console
+helm search repo boldbi
+
+NAME            CHART VERSION   APP VERSION     DESCRIPTION
+boldbi/boldbi   5.3.83           5.3.83         Embed powerful analytics inside your apps and t...
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Create Namespace
+
+Run the following command to create the namespace in which the Bold BI resources will be dpleoyed in the kubernetes cluster.The default namespace is <i>bold-services</i>
+
+```console
+kubectl create ns bold-services
+```
+
+## Install Chart
+
+For Helm chart, you'll need to craft a `values.yaml`file. So download the values.yaml file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/UMP-22952-Autodeployment-documentation/helm/custom-values/values.yaml) and make needed changes based on your cluster provider. 
+
+The below table helps you to craft he values.yaml file with required values for Bold BI deployment. so please read the describtion carfully and enter the values in values. yaml file.
```

Comment:
The following table allows you to craft the values.yaml file with required values for Bold BI deployment. so please read the description carefully and enter the values in values.yaml file.

Reply:
concern addressed